5.14.0/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM java:7-jre
+FROM java:8-jre
```

Comment:
could you remove everything except this line from this commit since we agreed to stay on java7 for the old images?